src/main/java/picocli/CommandLine.java

```diff
@@ -9166,9 +9175,25 @@ private String defaultValueFromProvider() {
              * @return whether this argument applies to all descendent subcommands of the command where it is defined
              * @since 4.3 */
             public ScopeType scopeType() { return scopeType; }
+            
+            /** Check whether the {@link #getValue()} method is able to get an actual value from the current {@link #getter()}. */
+            public boolean isValueGettable() {
+                if (getter instanceof IScoped) {
+                    IScoped scoped = (IScoped) getter;
+                    IScope scope = scoped.getScope();
+                    try {
+                        Object obj = scope.get();
+                        return obj != null;
```

Comment:
The `IScope scope` object itself (what is returned from `scoped.getScope()`) may also be `null`.
We have a try/catch around this, so any NPEs would be caught and handled correctly, but should we do this instead?

```java
return scope != null && scope.get() != null;
```

Reply:
Implemented all changes as requested